pkg/kubectl/describe.go

```diff
@@ -528,7 +529,30 @@ func describeContainers(pod *api.Pod, out io.Writer) {
 
 		fmt.Fprintf(out, "    Ready:\t%v\n", printBool(status.Ready))
 		fmt.Fprintf(out, "    Restart Count:\t%d\n", status.RestartCount)
+		fmt.Fprintf(out, "    Environment Variables:\t\n")
+		for _, e := range container.Env {
+			if e.ValueFrom != nil && e.ValueFrom.FieldRef != nil {
+				valueFrom := envValueFrom(pod, e)
+				fmt.Fprintf(out, "        \t%s\t = %s (%s:%s)\n", e.Name, valueFrom, e.ValueFrom.FieldRef.APIVersion, e.ValueFrom.FieldRef.FieldPath)
```

Comment:
Is it necessary to have the leading 8 spaces and then the tab? Wouldn't the 8 spaces be enough?